tools/docker/image-pull.sh

```diff
@@ -45,11 +46,7 @@ main() {
             -v | --verbose)         VERBOSE=true; shift ;;
             -h | --help)            usage; exit 0; shift ;;
             -b | --base-image)      IMAGE="$2"; shift ; shift ;;
-            -n | --native)          IMAGE=$(
-                                        . /etc/os-release
-                                        distro="$(echo $NAME | awk '{print tolower($0)}')"
-                                        echo "$distro:$VERSION_ID"
-                                    ); shift ;;
+            -n | --native)          NATIVE=true; shift; ;;
```

Comment:
Which shellcheck error is this trying to fix? That should be part of the commit message, ideally.

If it is just a refactoring because you think it's cleaner that way, do that in a separate commit. That way, it's easier to see what is really the shellcheck fix.

However, I don't think anybody uses the native option - it's an idea I had a long time ago but it doesn't really work. So instead, just remove the option.

Reply:
Add issues founded by shellcheck before refactoring into commit messages for all fixed scripts.
Create separate commit with removing "--native" option.